Google Cloud Pub/Sub gRPC: Akka Discovery (#1991)
akka/alpakka: docs/src/main/paradox/google-cloud-pub-sub-grpc.md

## Artifacts

> As Akka gRPC uses Akka Discovery internally, make sure to add Akka Discovery of the same Akka version as the application uses.

Reviewer: ```suggestion
Akka gRPC uses Akka Discovery internally. Make sure to add Akka Discovery with the same Akka version that the application uses.
```
